src/coreclr/build-test.cmd

```diff
@@ -280,8 +287,6 @@ if errorlevel 1 (
     exit /b 1
 )
 
-:skipnative
```

Comment:
Why is this removed?

Reply:
I believe it's not used anywhere. Perhaps I'm just blind, are you aware of any goto or call statement referring to this label?

Reply:
Just above at the top of this "section": "if defined __SkipNative goto skipnative"

Reply:
Hmm, my bad, thanks for pointing that out, for some reason I was under the impression I saw two instances of the label in the file. I probably need to buy myself stronger glasses :-). Will put back...